site/tutorials/keptn-multistage-qualitygates-09.md

```diff
@@ -49,14 +49,14 @@ The demo resources can be found on Github for a convenient experience. Let's clo
 
 <!-- command -->
 ```
-git clone https://github.com/cncf/podtato-head.git
+git clone https://github.com/keptn/examples --single-branch
```

Comment:
Please always refer to a release tag/branch, see https://github.com/keptn/tutorials/blob/master/site/tutorials/snippets/09/manage/createProject.md as an example.

Using the default branch of the keptn/examples repo introduces a risk that we will have an incompatible change at some point.